assets/.jshintrc

```diff
@@ -3,25 +3,33 @@
   "browser": true,
   "esnext": true,
   "bitwise": true,
-  "camelcase": true,
+  "camelcase": false,
   "curly": true,
   "eqeqeq": true,
   "immed": true,
   "indent": 2,
   "latedef": true,
-  "newcap": true,
+  "newcap": false,
   "noarg": true,
+  "quotmark": false,
   "regexp": true,
-  "undef": true,
-  "unused": true,
+  "smarttabs": true,
   "strict": true,
+  "sub" : true,
   "trailing": true,
```

Comment:
does jshint still recognize this as an option? it's fine to set your editor to trim it, but I'd rather not enforce it...